<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CR5SalaryByWorkersFilter</class>
 <widget class="QWidget" name="CR5SalaryByWorkersFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="C5DateEdit" name="deStart">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="FilterName" stdset="0">
      <string>date1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="C5DateEdit" name="deEnd">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="FilterName" stdset="0">
      <string>date2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Employee</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="C5LineEditWithSelector" name="leEmployee">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="C5LineEditWithSelector" name="leEmployeeName">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>End</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Shift</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="C5LineEditWithSelector" name="leShift">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="C5LineEditWithSelector" name="leShiftName">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEditWithSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineeditwithselector.h</header>
  </customwidget>
  <customwidget>
   <class>C5DateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5dateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
